package-lock.json

```diff
@@ -34880,7 +34880,7 @@
       "license": "Apache-2.0",
       "dependencies": {
         "@babel/runtime": "^7.17.9",
-        "indigo-ketcher": "1.22.0-rc.3",
+        "indigo-ketcher": "^1.23.0-dev.4",
```

Comment:
Let's keep version without "^" to have consistent version of indigo for all developers.